packages/lib-classifier/src/plugins/tasks/dropdown-simple/components/SimpleDropdownTask.js

```diff
@@ -103,26 +89,24 @@ function SimpleDropdownTask({
       <Box
         gap='xsmall'
       >
-        <Select
+        <input
+          list="simple-dropdown-list"
+          id="simple-dropdown-choice"
```

Comment:
Having a fixed ID means this component can only be used once on the page.

Reply:
Yup - temporary for POC. Fixed.